### ISSUE TITLE
Feature Idea: Image preview/download for attached images

### ISSUE THREAD

**Issue by N3rdP1um23** [mattermost/focalboard#3475]
## Summary

When pasting/adding images into a card, there's no way to preview that image or even download it. Due to this nature, it can be hard to review attached images or even download them to preview them in another application.

## How important this is to me and why

Importance: High

Use cases:
1. When an image is attached to a card as content, when clicking on the image, it should open up another modal with the image preview
2. From there, the user could then be able to view the image in full size or "fitted" to their screen size and allow them to zoom
3. In the preview they could even download the image for a number of reasons

1. When clicking on the three dots beside an image, it would be nice to have a download option - or even implement one that displays when hovering over images to save a few clicks

## Additional context/similar features

A project I'm currently working on requires sharing of mockup images between designers and developers. Unfortunately, while we can attach images, there's no way for the developers to "enhance" the image in the card to view the full design. This has left us with having to share images outside of the card and including a link to that external source. It would be nice to have it as an all-in-one system to remove the need for an external dependency. Other than that, it's been great!!

**Comment by andyylin** [comment]
Yes, very much agree. Our organization just started using Mattermost and this seems like a critical feature for cards.

**Comment by andyylin** [comment]
Seems same as #2364

**Comment by igmarketing** [comment]
Wow this has been requested since 2022, and still not available. I know Focalboard development got some difficulties recently but that would be amazing if a contributor could implement this feature... 

It's a deal-breaker actually... this ios a must-have feature. I share often screenshots with my dev and they are in 1080p... and its impossible to see anything in the minimized image preview.
